### PR TITLE
#160534559 create endpoint to update order status

### PR DESCRIPTION
**Pull request by nwashangai**
#### What does this PR do?
it creates an endpoint to update order status
#### Description of Task to be completed?
* add a route to the API to update user order status
#### How should this be manually tested?
* Not applicable
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
#159760849
#### Screenshots (if appropriate)
* None
#### Questions:
* None

### CODE REVIEW

File: src/utils/updateOrder.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { order } from '../models/store';
+
+exports.updateOrder = (id, status) => {
```
Comment (dogoyaro) [inline src/utils/updateOrder.js]: Please look at the right export method.

File: test/order.js [hunk 2]
```diff
@@ -68,5 +68,41 @@ describe('Fast-Food-Fast', () => {
           done();
         });
     });
+    it('it should update order with ID 3', (done) => {
```
Comment (dogoyaro) [inline test/order.js]: Use a generic title for your test-case descriptions. `it should successfully update order given an Id` seems like a superior title to `it should update order with ID 3`.

**Review by dogoyaro — CHANGES_REQUESTED** [review]

File: test/order.js (hunk 2 above)
Comment (nwashangai) [reply test/order.js]: Ok this commit has been fixed

File: src/utils/updateOrder.js (hunk 1 above)
Comment (nwashangai) [reply src/utils/updateOrder.js]: Ok fixed

File: src/config/index.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-module.exports = {
+export default {
   development: {
     // DATABASE_URL: 'postgres://postgres:andela@localhost:5432/myDiary',
```
Comment (dogoyaro) [inline src/config/index.js]: remove commented code.

**Review by dogoyaro — APPROVED** [review]

**Pull request merged**